docs/Developer Documentation/Web Engine Support/index.md

```diff
@@ -0,0 +1,101 @@
+# Web Engine Application Guide
```

Comment:
All proposals about this feature use "WebEngine" - can this be changed throughout?